app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>
+  <strong>Commenter:</strong>
+  <%= comment.commenter %>
+</p>
+
+<p>
+  <strong>Comment:</strong>
+  <%= comment.body %>
+</p>
+
+<p>
+  <%= link_to 'Destroy Comment', [comment.article, comment], method: :delete, data: { confirm: "Are you sure?" } %>
+</p>
```

Comment:
Remember single quotes and the empty line at the end of the file.

Reply:
Done